docs: example how to verify call arguments through asymmetric matchers (#13787)
facebook/jest: docs/ExpectAPI.md

});
```

> Note that you can use any asymmetric matchers to validate specific arguments, combining it with literal values:
> 
> ```
> expect(func).toHaveBeenCalledWith(
>   expect.anything(),
>   expect.any(MyClass1),
>   42,
>   expect.objectContaining({
>     name: expect.stringContaining('John')
>   }),
>   false
> );
> ```
> 

Reviewer: Perhaps:

````suggestion
:::tip

You can also use asymmetric matchers instead of literal values:

```js
expect(func).toHaveBeenCalledWith(
  expect.anything(),
  expect.any(MyClass),
  42,
  expect.objectContaining({
    name: expect.stringContaining('John')
  }),
  false
);
```

:::
````
